website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx

```diff
@@ -44,10 +45,12 @@ To delete a key, find it in the list of keys and click its "Delete" button. Befo
 
 To assign a key to a workspace, navigate to that workspace's page and choose "SSH Key" from the "Settings" menu.
 
+**SCREENSHOT FOR DELETION**
 ![Terraform Cloud screenshot: the settings menu on a workspace's page](/img/docs/settings-tabs.png)
```

Comment:
hmmm i don't see this on the actual page that's live right now --> maybe this is me though and I'm missing something!
https://developer.hashicorp.com/terraform/cloud-docs/workspaces/settings/ssh-keys